en/docs/learn/consume-api/manage-subscription/advanced-topics/adding-an-api-subscription-tier-update-workflow.md

```diff
@@ -25,12 +25,12 @@ When the API subscription update workflow is enabled, when the subscription tier
 
 3.  Sign in to the WSO2 API Developer Portal (`https://<hostname>:<port>/devportal`) and click **Applications**. Select the application which has the subscriptions you wish to modify.
 
-    ![[applications overview_tab]({{base_path}}/assets/img/learn/application-overview.png)]
+    ![[applications overview_tab]({{base_path}}/assets/img/learn/application-overview.png)
 
 
-4. Click **Subscriptions** to list the subscriptions of the application.
+4.  Click **Subscriptions** to list the subscriptions of the application.
     
-   ![[subscriptions overview_tab]({{base_path}}/assets/img/learn/subscriptions-overview-tab.png)]
+    ![[subscriptions overview_tab]({{base_path}}/assets/img/learn/subscriptions-overview-tab.png)
```

Comment:
```suggestion
    [![subscriptions overview_tab]({{base_path}}/assets/img/learn/subscriptions-overview-tab.png)]({{base_path}}/assets/img/learn/subscriptions-overview-tab.png)
```

Reply:
@samgnaniah , the syntax of the images were wrong. Also always pls make the images zoomable.